frontend/components/wallet/WalletTable.vue

```diff
@@ -64,13 +64,13 @@
 <script lang="ts">
 import Vue from 'vue';
 import { WalletBalances } from 'auctions-core/src/types';
-import FormatAddress from '../utils/FormatAddress.vue';
-import FormatCurrency from '../utils/FormatCurrency.vue';
-import TimeTill from '../common/TimeTill.vue';
-import Loading from '../common/Loading.vue';
-import BaseButton from '../common/BaseButton.vue';
-import TextBlock from '../common/TextBlock.vue';
-import WalletBlock from '../transaction/WalletBlock.vue';
+import FormatAddress from '../common/formatters/FormatAddress.vue';
+import FormatCurrency from '../common/formatters/FormatCurrency.vue';
+import TimeTill from '../common/formatters/TimeTill.vue';
+import Loading from '../common/other/Loading.vue';
```

Comment:
Please use absolute imports where possible (except only for stories, where you already fixed the opposite 😉)

Reply:
Done! I went through every component and double checked that all were done via absolute imports!